config/routes.rb

```diff
@@ -2,5 +2,11 @@
   devise_for :users, controllers: {
     registrations: 'users/registrations'
   }
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  namespace :account do
+    resources :users, only: [:index, :show]
+    namespace :admin do
+      resources :organizations, only: [:index, :show]
+    end
+  end
```

Comment:
це трохи не те, з цим не буде зручно працювати просто
```suggestion
  namespace :account do
    resources :users, only: [:index, :show]
  end
  
  namespace :admin do
    resources :organizations, only: [:index, :show]
  end
```